### PR TITLE
[ADD] Added a gamification_badge_report module.

### PR DESCRIPTION
**Pull request by AmmarOfficewalaSerpentCS**

### CODE REVIEW

**Comment by AmmarOfficewalaSerpentCS** [comment]
@max3903 , @dreispt , 
Can you please review this PR?

File: gamification_badge_report/README.rst [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+.. |badge3| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/hr-12-0/hr-12-0-gamification_badge_report
+    :alt: Translate me on Weblate
+.. |badge4| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/116/12.0
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| 
+
+Display a total badge report gorup by with a employee.
```
Comment (marcelsavegnago) [inline gamification_badge_report/README.rst]:
```suggestion
Display a total badge report group by with a employee.
```

File: gamification_badge_report/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+Display a total badge report gorup by with a employee.
```
Comment (marcelsavegnago) [inline gamification_badge_report/readme/DESCRIPTION.rst]:
```suggestion
Display a total badge report group by with a employee.
```

**Review by marcelsavegnago — APPROVED** [review]
LGTM

**Pull request closed without merging**